alnair-device-plugin/build/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:stretch-slim
```

Comment:
we can create the build directory in the Makefile and put the Dockerfile in upper level directory?